packages/components/ContextualMenu/src/SubmenuItem.tsx

```diff
@@ -122,38 +122,38 @@ export const SubmenuItem = compose<SubmenuItemType>({
   settings,
   render: (Slots: ISlots<SubmenuItemSlotProps>, renderData: SubmenuItemRenderData, ...children: React.ReactNode[]) => {
     const xml = `
-    <svg width="12" height="12" viewBox="0 0 2048 2048" >
+    <svg width="12" height="12" viewBox="0 0 2048 2048">
       <path class='OfficeIconColors_HighContrast' fill='currentColor' d='M 743 1767 l -121 -121 l 708 -707 l -708 -708 l 121 -121 l 828 829 z' />
-      <path class='OfficeIconColors_m22' fill='currentColor' d='M 743 1767 l -121 -121 l 708 -707 l -708 -708 l 121 -121 l 828 829 z' />
+      <path class='OfficeIconColors_m22'          fill='currentColor' d='M 743 1767 l -121 -121 l 708 -707 l -708 -708 l 121 -121 l 828 829 z' />
```

Comment:
nit: Why the extra spaces here?

Reply:
Uh, in vscode it lines up the two svg elements and makes it more obvious they're the same path. Speaking of, @valentinakozlova why are these the same path?

Reply:
@Saadnajmi , I don't know. I didn't add it for SubmenuItem. @lenahong , do you know why is there the same path?

Reply:
Right, sorry, You did it for MenuButton, my bad =)